### PR TITLE
Correcting the tag names of the EventHub split swagger

### PR DESCRIPTION
**Pull request by mythri1006**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [ ] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters": {
       "get": {
         "tags": [
-          "Event Hubs Clusters"
+          "Clusters"
```
Comment (ppgovekar) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json]: NIT: I will suggest to keep "Event Hubs Clusters" as it is, "Clusters" means something different in Azure terminology

Comment (v-Ajnava) [reply specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json]: @mythri1006 can you please make this change ?

Comment (mythri1006) [reply specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json]: Reverted the tag name changes as informed above

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/virtualnetworkrules-preview.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListVirtualNetworkRules",
+        "operationId": "VirtualNetworkRules",
```
Comment (v-Ajnava) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/virtualnetworkrules-preview.json:42]: change should be in tags, please revert the operationId change

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json [hunk 3]
```diff
@@ -79,7 +79,7 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}": {
       "get": {
         "tags": [
-          "Event Hubs Cluster"
+          "Event Hubs Clusters"
```
Comment (v-Ajnava) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json:82]: change to Event Hubs Cluster

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/AvailableClusterRegions-preview.json [hunk 4]
```diff
@@ -37,7 +37,7 @@
     "/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/availableClusterRegions": {
       "get": {
         "tags": [
-          "Event Hubs Clusters"
+          "AvailableClusterRegions"
```
Comment (v-Ajnava) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/AvailableClusterRegions-preview.json:40]: change to Event Hubs Cluster

Comment (v-Ajnava) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/AvailableClusterRegions-preview.json:40]: change it to "Event Hubs Cluster"

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json (hunk 3 above)
Comment (v-Ajnava) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json:82]: change it to "Event Hubs Cluster"

**Review by v-Ajnava — CHANGES_REQUESTED** [review]

**Comment by v-Ajnava** [comment]
@mythri1006 there are other tags missing from the above files, please recheck and make those changes

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json (hunk 3 above)
Comment (v-Ajnava) [reply specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json:82]: or make "Event Hubs Clusters" everywhere

**Pull request closed without merging**